lib/companies_web/templates/user/card.html.eex

```diff
@@ -0,0 +1,16 @@
+<div class="company box %>">
+  <div class="content has-text-centered">
+    <p class="title is-4 thin"><%= link to: Routes.user_path(@conn, :show, locale(@conn), @user) do %><%= Map.get(@user, :name) %><% end %></p>
```

Comment:
This route ` Routes.user_path(@conn, :show, locale(@conn), @user)` is not implemented right?

Do we want to expose a public route for users who are currently in the database? Or only for those who have said they are looking for a job.

Reply:
Only for those who are looking for a job. This is still WIP, lot's of work to be done.